Makefile

```diff
@@ -88,8 +88,6 @@ test:
 	@pushd ${radish34} && \
 	npm run await-stack && \
 	npm run test && \
-	npm run postman-test-zkp-api && \
```

Comment:
I see you've deleted both postman test execution steps here but in the package.json only removed one of the tests. Are you combining them? How will they run as part of `make test` and/or `npm`?

Reply:
The one I deleted is the redundant integration-level test suite. The other one is a test suite specifically for the zkp service. I think all service-specific test suites should be run by `npm run test` from within the individual service. So that npm script should be moved into `zkp/package.json`. Do you agree?

Reply:
I moved the script into `zkp/package.json` to conform to structure in other service directories